department-app/tests/__main__.py

```diff
@@ -0,0 +1,3 @@
+"""tests for the app"""
+
+print("hello, coveralls")
```

Comment:
This file is redundant and can be removed.

Reply:
done